public/app/api/clients/provisioning/v0alpha1/endpoints.gen.ts

```diff
@@ -25,6 +25,43 @@ const injectedRtkApi = api
         }),
         providesTags: ['Job'],
       }),
+      createJob: build.mutation<CreateJobApiResponse, CreateJobApiArg>({
```

Comment:
@Clarity-89 -- is there a way to have the client exclude all the write methods?  These exist, but should only ever be called by backend services